### PR TITLE
Some fixes and updates

### PR DESCRIPTION
**Pull request by robsonsanches**
Hello @masasron 

I made some adjustments to the functions that made the checkboxes checked.
Also organized the files into separate folders "src" and "dist".
I added gulp to the project to be able to automate the tasks of minimizing files.
I believe that these adjustments and updates will further improve your plugin which is already very cool.

Best regards

### CODE REVIEW

File: dist/js/iosCheckbox.js [hunk 1]
```diff
@@ -52,10 +52,10 @@
                     // Check if the ios checkbox is checked
                     if (ios_checkbox.hasClass("checked")) {
                         // Update state
-                        org_checkbox.prop('checked', true);
+                        org_checkbox.attr('checked','checked');
```
Comment (robsonsanches) [inline dist/js/iosCheckbox.js:69]: This solves a problem that prevented the checkbox type input from getting checked status when clicking the button.

File: dist/js/iosCheckbox.js [hunk 2]
```diff
@@ -57,9 +71,6 @@
                         // Update state
                         org_checkbox.removeAttr('checked');
                     }
-                    
-                    // Run click even in case it was registered to the original checkbox element.
-                	org_checkbox.click();
```
Comment (robsonsanches) [inline dist/js/iosCheckbox.js:62]: Removed code snippet as it was leaving the button active and the checkbox field inactive (unchecked)

Comment (masasron) [reply dist/js/iosCheckbox.js:62]: 👍

File: dist/js/iosCheckbox.js (hunk 1 above)
Comment (masasron) [reply dist/js/iosCheckbox.js:69]: 👍

**Comment by masasron** [comment]
Looking good @robssanches 👍 

Thank you!
Ron

**Comment by robsonsanches** [comment]
> 
> 
> Looking good @robssanches 👍
> 
> Thank you!
> Ron

You are welcome

**Pull request merged**